src/transaction.rs

```diff
@@ -1274,9 +1274,9 @@ mod tests {
                 genesis_hash: bitcoin::BlockHash::from_hex(
                     "0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206"
                 ).unwrap(),
-                script_pubkey: hex_deserialize!(
-                    "1976a914bedb324be05d1a1254afeb3e7ef40fea0368bc1e88ac"
-                ),
+                script_pubkey: bitcoin::Script::from_hex(
+                    "76a914bedb324be05d1a1254afeb3e7ef40fea0368bc1e88ac"
```

Comment:
The `0x19` length prefix was needed for `encode::deserialize()`, but isn't when directly constructing a `Script`.